modules/tfm/tfm/boards/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@
 
 set(NRF_BOARD_SELECTED True)
 
+set(nrf_dir ${CMAKE_CURRENT_LIST_DIR}/../../../../)
```

Comment:
Question, don't we have access to https://github.com/nrfconnect/sdk-nrf/blob/dd6b5859753197829f56372ef750b0b76c4e0155/CMakeLists.txt#L8 here?

Reply:
No, this is built in the TFM build system

Reply:
then please pass the NRF_DIR or ZEPHYR_NRF_MODULE_DIR to TF-M build system.
This can be done by appending settings to TFM_CMAKE_OPTIONS on the zephyr_property_target.

Reply:
done